config/prow/config.yaml

```
@@ -535,6 +535,29 @@ tide:
     - needs-rebase
   - repos:
     - kubernetes/kubernetes
+    excludedBranches:
+    - master
```

```suggestion
    - master
    - release-1.22
```
I just noticed a little error here, we have to add the 1.22 branch (`release-1.22`) to this exclusion and also to the requirements below. The branch does not exist yet, but we want code freeze to also apply to it when we create it in `v1.22.0-rc.0`.